.github/workflows/create-pr-compose-release.yml

```diff
@@ -0,0 +1,42 @@
+name: Create a automatic PR on Compose Release
+description: create a Pull Request with doc changes when a new version of Docker Compose is released
+
+on:
+  repository_dispatch:
+    types: [compose_release]
+
+jobs:
+  create_compose_doc_pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout docs source code
+        uses: actions/checkout@v3
+
+      - name: checkout Compose source code
+        with:
+          repository: docker/compose
+          ref: ${{ github.event.inputs.release }}
+          path: ./ComposeReleaseSource
+      - name: Prepare PR
+        run: |
+          rm -rf compose/reference/*
+          cp -rf /.ComposeReleaseSource/docs/reference/* ./compose/reference/
+          rm -rf /.ComposeReleaseSource
+          git add -A .
+      - name: Create PR
+        uses: peter-evans/create-pull-request@v4
+        with:
+          commit-message: Update Compose reference API documentation to ${{ github.event.inputs.release }}
+          signoff: true
+          branch: update-compose-doc-${{ github.event.inputs.release }}
+          delete-branch: true
+          title: Update Compose reference API doc - ${{ github.event.inputs.release }}
```

Comment:
@glours Is this CLI reference or API reference docs?

Reply:
This one https://github.com/docker/compose/tree/v2/docs/reference